packages/mdx-embed/src/components/youtube/youtube.tsx

```diff
@@ -41,6 +41,7 @@ export const YouTube: FunctionComponent<IYouTubeProps> = ({
         }}
       >
         <iframe
+          data-testid={`youTube-${youTubeId}`}
```

Comment:
The `data-testid` is only required for unit testing.

Reply:
My bad, I pushed the wrong bit of code! 🤦‍♂️

Reply:
no worries man! 😎